arm_link1_controller:
  type: "effort_controllers/JointPositionController"
  joint: "arm_joint1"
  pid: {p: 10.0, i: 0.01, d: 1.0}

arm_link2_controller:
  type: "effort_controllers/JointPositionController"
  joint: "arm_joint2"
  pid: {p: 10.0, i: 0.01, d: 1.0}